exercise/cache/content.py

```diff
@@ -183,6 +185,43 @@ def add_to(target: Dict[str, Any], exercise: BaseExercise) -> None:
         if total['min_group_size'] > total['max_group_size']:
             total['min_group_size'] = 1
 
+        exercises = []
+        for module in modules:
+            #add the module fields to cached exercise endpoint
+            api_url = f"{settings.BASE_URL}/api/v2/courses/{instance.id}/exercises/{module['id']}/"
```

Comment:
Use the `api_reverse` function to build URLs.

https://github.com/apluslms/a-plus/blob/d7dc06549856b10306732a892735cb0d9d66ed0b/aplus/api/__init__.py#L4